packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -118,6 +118,20 @@ new rds.DatabaseClusterFromSnapshot(this, 'Database', {
 });
 ```
 
+By default, automatic minor version upgrades for the engine type are enabled in a cluster, but you can also disable this.
```

Comment:
CFN docs for reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-autominorversionupgrade